src/Entity/SkillTrait.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+trait SkillTrait
+{
+    public function addSkill(Skill $skill): void
```

Comment:
Not sure about that one, this method used to call the reverse side of the ManyToMany.